### DISCUSSION TITLE
Ability to allow access to all entries (including new ones) except a subset

### DISCUSSION THREAD

**Discussion by GaryReckard** [trendyminds/isolate#40] [Ideas]
From what I can tell, a user can be allowed to edit all entries in a section, or you can choose to allow a certain subset of entries. For my use case, we want to allow the user to edit all entries except a small subset. If I select all entries and uncheck the small subset, this works fine, until a new entry is added to that section. It looks like that user would NOT be able to edit that new entry, but I'd like them to be able to .

Would it be possible to have a setting that makes any new entries by default allowed for a particular user/section?

Or another approach would be to offer an alternate interface, where instead of explicitly selecting all entries the user can edit, you would be explicitly selecting the entries they CANNOT edit (implicitly, all other entries would be editable).

Thanks!

**Comment by aaronbushnell** [comment]
Hello! We've moved all open Issues to **Discussions**.

Unfortunately, due to client workload we often find ourselves unable to implement new features or resolve issues in a timely manner. Switching to Discussions allows better categorization to separate "Feature Requests" from bugs. It also allows the community to better suggest/provide solutions when we're unable to do so.

Hope Isolate continues to be useful for you!
